WireAuthentication/Sources/WireAuthenticationUI/Views/Root/RootViewModel.swift

```diff
@@ -36,4 +44,12 @@ public final class RootViewModel: ObservableObject, Router {
         path.append(destination)
     }
 
+    public func presentSheet(_ modalDestination: some Hashable) {
+        self.modalDestination = modalDestination as? RootView.ModalDestination
+    }
+
+    public func presentAlert(_ alert: some Hashable) {
+        self.alert = alert as? RootViewModel.Alert
+    }
```

Comment:
non-blocking: I think the `some Hashable` isn't needed because we're not appending to a navigation path.